llvm/include/llvm/CodeGen/ScheduleDAGInstrs.h

```diff
@@ -124,6 +124,9 @@ namespace llvm {
     /// rescheduling).
     bool RemoveKillFlags;
 
+    /// True if regions with a single MI should be scheduled.
+    bool ScheduleSingleMIRegions = false;
```

Comment:
Over-configuration? Can you just unconditionally do this?

Reply:
I removed the flag and made this the generic scheduler behavior. The only unit test that breaks in a way I cannot really understand is `misched-branch-targets.mir`. I "fixed" it anyway, but not sure if the change break something here.